elfi/methods/mcmc.py

```diff
@@ -387,6 +387,8 @@ def metropolis(n_samples, params0, target, sigma_proposals, seed=0):
         The target log density to sample (possibly unnormalized).
     sigma_proposals : np.array
         Standard deviations for Gaussian proposals of each parameter.
+    warmup : int
+        Initial number of samples to be thrown away
```

Comment:
I think this implies that the warmup is included in `n_samples`, but the current code does `n_samples + warmup` iterations.

Reply:
Fair point. I'll change the wording.